utils/queryPathResolver.ts

```diff
@@ -34,7 +34,7 @@ function getPath(prefix: string) {
     case 'subsquid':
       return 'subsquid/general/'
     default:
-      return ''
+      return 'subsquid/general/'
```

Comment:
I feel that this line might break something

Reply:
This looks like it will break Magick queries

Reply:
Thanks for info Leo ❤️ fixed in ed41b76040b2c76fa9b2a3a32083282547e6c9a8